lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -46,6 +46,16 @@
     description: The provider region name to connect to (e.g. AWS region for Amazon).
     required: false
     default: null
+  host_default_vnc_port_start:
+    required: false
+    default: null
+    description: The first port in the host VNC range. defaults to None.
+    version_added: "2.5"
+  host_default_vnc_port_end:
+    required: false
```

Comment:
what happens if user specifies only one (e.g. end without start)?

Reply:
yeah, i think a required_together should do it. Would you mind if I submitted that as a follow-up patch as I have removed my old repo this commit comes from after getting in a git mess.

Reply:
Something like:

```
required_together = (['host_default_vnc_port_start', 'host_default_vnc_port_end'])
```

Reply:
sure, that's fine